react-select/react-select.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for React-Select v1.0.0-beta
+// Project: https://github.com/JedWatson/react-select
+// Definitions by: Roger Chen <https://github.com/rcchen>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../react/react.d.ts" />
+
+declare module "react-select" {
+    import { Component } from 'react';
+
+    interface IReactSelectOption {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.